backend/README.md

```diff
@@ -18,4 +18,5 @@ If you require any assistance or support, feel free to let us know. Join our Sla
 Ohha! The participation list is totally empty. Hurry up if your want to have your name
```

Comment:
```suggestion

```

Reply:
You are right, I will do that right away